CONTRIBUTING.md

```diff
@@ -21,27 +59,11 @@ There is a built-in commit linter. Basic rules:
 * Commit messages **must** start with a capital letter
 * Commit messages **must not** end with a period `.`
 
```

Comment:
I think we should mention our testing requirements and how to fix common linting errors.

Reply:
I added a step in the contributing guide to run tests before committing work. `npm test` runs the formatter and linter already. The linter will output the lint errors so im confused what you meant by "how to fix common linting errors". 

Also, I didnt know we had testing requirements. Did you mean PRs must include tests or requirements as far as structuring tests when they are written?

Reply:
Adding `npm test` is good but it doesn't do the fixes it only fails when some issue is found.  To fix formatting issues it would be `npm run format`

An example when someone makes a suggested edit and the author accepts the edit the formatting of the code usually fails the linter check.

Another example 
adding or updating html sometimes makes the snapshots fail so `npm run update-snapshot` will update the snapshots

Reply:
> Also, I didnt know we had testing requirements. Did you mean PRs must include tests or requirements as far as structuring tests when they are written?
I just mean the current tests must still pass and the ci and linting must pass. Ideally new features would include tests but I dont think it should block a PR.

Reply:
Ahh good point. I’ll add those in